### DISCUSSION TITLE
MAUI SelectTabAsync

### DISCUSSION THREAD

**Discussion by pierre-galaup** [PrismLibrary/Prism#2841] [.NET MAUI]
Hello,

I am migrating my application from XamarinForms to MAUI.

I am using a TabbedPage and the PRISM FORMS method: 

`navigationService.SelectTabAsync(nameof(MyTabPage), navigationParameters)`

I can't find an equivalent in MAUI PRISM, if I do a :

`await _navigationService.NavigateAsync("TabbedPage?selectedTab=MyTabPage");`

This re-launches a TabbedPage.

Can you help me with this?

Thank you,
Pierre

**Comment by giuseppenovielli** [comment]
Yes +1, see my examples from Xamarin.Forms 

—>https://github.com/giuseppenovielli/MAUI_XF-Prism-Full-Navigation-Example

I can’t select tab into MAUI.

@dansiegel @brianlagunas please Help.

Thanks!!

**Comment by AnthonyLatty** [comment]
Any update on this?

**Reply by dansiegel** [reply]
I do take PRs if it's important to you guys

**Reply by LeoJHarris** [reply]
@dansiegel I'm guessing this is still not supported?

**Reply by dansiegel** [reply]
it's been merged (#3014) and will be in the next release... it's currently available to those with the Commercial Plus License.
